include/nbl/asset/IBufferView.h

```diff
@@ -19,7 +19,7 @@ template<typename BufferType>
 class IBufferView : public IDescriptor
 {
 	public:
-		_NBL_STATIC_INLINE_CONSTEXPR size_t whole_buffer = ~static_cast<size_t>(0u);
+		static inline constexpr size_t whole_buffer = ~static_cast<size_t>(0u);
```

Comment:
please use the provided macro

Reply:
theres no point once we moved to cpp17

Reply:
ok so MSVC no longer complains?

Reply:
it complained something like "available since cpp17" before

Reply:
ok cool